api/GLES3/gl31.h

```diff
@@ -1,12 +1,12 @@
-#ifndef __gl31_h_
-#define __gl31_h_ 1
+#ifndef __gles2_gl31_h_
```

Comment:
Did you re-run genheaders.py against these targets specifically, or was this a manual edit? They were taken out from the Makefile GLHEADERS target at some point, probably because they are out of date versions of the API, but no reason not to regenerate them. The edits seem fine to me but I'd like to get signoff from @pdaniell-nv before merging.

Reply:
Huh, bizarre. gl31.h isn't one of the targets, AFAICT. But harmless.